src/components/interactive-builder/interactive-builder.component.tsx

```diff
@@ -185,22 +299,43 @@ const InteractiveBuilder: React.FC<InteractiveBuilderProps> = ({
                 ) : null}
                 {page?.sections?.length ? (
                   page.sections?.map((section, sectionIndex) => (
-                    <Accordion className={styles.accordion}>
-                      <AccordionItem title={section.label}>
```

Comment:
@denniskigen Does the css in these div replicate the behaviour of the accordion? Long forms benefited from expanding and collapsing sections.

Reply:
It doesn't, unfortunately. I haven't thought of a better UI for editing section labels when using Accordions. One way would be to have the section's label as an editable field (alongside its related questions) within an Accordion item.

Reply:
Accordions are back